brave/brave-core
# feat(wallet): Add caching for NFT metadata response

Resolves https://github.com/brave/brave-browser/issues/30383



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Open an NFT from NFT tab
2. Open developer console and go to `Application` tab
3. Under storage, select localStorage. There should be a key `BRAVE_WALLET_NFT_METADATA_CACHE` containing NFT metadata response
